Packs/Illumio/Integrations/IllumioCore/IllumioCore.py

```diff
@@ -0,0 +1,1232 @@
+    results = json.loads(json.dumps(response, cls=IllumioEncoder))
+    context_data = {}
+    context_data["hrefs"] = [service.get("href", "") for service in results.get("service_bindings", [])]
+    readable_output = prepare_service_binding_output(results)
+
+    return CommandResults(
+        outputs_prefix="Illumio.ServiceBinding",
+        readable_output=readable_output,
+        outputs_key_field="href",
+        outputs=remove_empty_elements(context_data),
+        raw_response=results,
+    )
+
+
+def object_provision_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> CommandResults:
+    """
+    Command function for illumio-objects-provision command.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: arguments obtained from demisto.args()
+
+    Returns:
+        Standard command results.
+    """
+    security_policy_objects = args.get("security_policy_objects", "")
+    validate_required_parameters(security_policy_objects=security_policy_objects)
+    security_policy_objects = argToList(security_policy_objects)
+    change_description = generate_change_description_for_object_provision(
+        hrefs=security_policy_objects
+    )
+    response_dict = {}
+    try:
+        response_object = client.provision_policy_changes(
+            change_description=change_description, hrefs=security_policy_objects
+        )
+        response_dict = response_object.to_json()
+        hr_output = prepare_object_provision_output(response_dict)
+
+        # Converting draft HREFs to active
+        provisioned_hrefs = [
+            convert_draft_href_to_active(href) for href in security_policy_objects
+        ]
+
+        response_dict["provisioned_hrefs"] = provisioned_hrefs
+    except Exception as e:
+        if EXISTING_OBJECT not in str(e):
+            raise Exception("Encountered error while provisioning security policy object: {}".format(e))
+        else:
+            hr_output = "### Security policy object(s) already provisioned: {}.".format(
+                ", ".join(security_policy_objects))
+
+    return CommandResults(
+        outputs_prefix="Illumio.PolicyState",
+        outputs_key_field="href",
+        outputs=remove_empty_elements(response_dict),
+        readable_output=hr_output,
+        raw_response=response_dict,
+    )
+
+
+def workload_get_command(client: PolicyComputeEngine, args: Dict[str, Any]) -> CommandResults:
+    """Retrieve a workload.
+
+    Args:
+        client: PolicyComputeEngine to use.
+        args: arguments obtained from demisto.args()
+
+    Returns:
+        CommandResult object
+    """
+    href = args.get("href")
+    validate_required_parameters(href=href)
+
+    response = client.workloads.get_by_reference(href)  # type: ignore
+    results = json.loads(json.dumps(response, cls=IllumioEncoder))
+
+    readable_output = prepare_workload_get_output(results)
+
+    return CommandResults(
+        outputs_prefix="Illumio.Workload",
```

Comment:
```suggestion
        outputs_prefix="Illumio.Workloads",
```
take into account that the command might run multiple times and it might contain multiple results (yiu already have a path named `Illumio.Workloads"`
please fix it in the yml and readme files as well.

Reply:
Make sense, updated the same.